sdk/nodejs/dagger/dist/engine.js

```diff
@@ -38,9 +38,23 @@ export class Engine {
             this.config.Port = this.config.Port || 8080;
             args.push("--port", `${this.config.Port}`);
             const serverProc = execa("dagger", args, {
-                stdio: "inherit",
+                stdin: "inherit",
+                stdout: "inherit",
+                stderr: "pipe",
                 cwd: this.config.Workdir,
             });
+            serverProc.catch(err => {
+                let error = err;
```

Comment:
nit: any particular reason of declaring `error` line 47 instead of using err directly ?